Misc/NEWS.d/next/Documentation/2024-01-25-15-58-31.gh-issue-114494.sgtopX.rst

```diff
@@ -0,0 +1 @@
+I updated the logging docstring to use bool instead of integer parameters for exec_info. Originally, it was written as exec_info=1 in 6 places, which I changed to exec_info=True.
```

Comment:
Don't state what you have done. State what change is affecting the user.

Example:
Minor fixes for ``logging`` docstring.

Reply:
remove this NEWS